CoreLayersView/src/au/gov/asd/tac/constellation/views/layers/LayersViewShortcuts.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2010-2020 Australian Signals Directorate
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package au.gov.asd.tac.constellation.views.layers;
+
+import au.gov.asd.tac.constellation.graph.manager.GraphManager;
+import au.gov.asd.tac.constellation.plugins.PluginExecution;
+import au.gov.asd.tac.constellation.views.layers.utilities.DeselectAllLayersPlugin;
+import au.gov.asd.tac.constellation.views.layers.utilities.NewLayerPlugin;
+import java.awt.event.ActionEvent;
+import javax.swing.AbstractAction;
+import org.openide.awt.ActionID;
+import org.openide.awt.ActionReference;
+import org.openide.awt.ActionReferences;
+import org.openide.awt.ActionRegistration;
+import org.openide.util.NbBundle;
+
+/**
+ * Defines the keyboard shortcuts for use with the Layers view
+ *
+ * @author formalhaut69
+ */
+@ActionID(category = "Options", id = "au.gov.asd.tac.constellation.views.layersviewshortcuts")
+@ActionRegistration(displayName = "#CTL_LayersViewShortcuts", surviveFocusChange = true)
+@ActionReferences({
+    @ActionReference(path = "Shortcuts", name = "CA-S"),
```

Comment:
This probably should have been changed to "CA-D"

Reply:
I'm not sure what happened here, I made that change here; https://github.com/constellation-app/constellation/pull/782/commits/3ba3d12396b74588c329b523fde94d6025e61d1e